modules/virt-define-tcp-readiness-probe.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-monitoring-vm-health.adoc
+
+[id="virt-define-tcp-readiness-probe_{context}"]
+
+= Defining a TCP readiness probe
+
+Define a TCP readiness probe by setting the `spec.readinessProbe.tcpSocket` field of the virtual machine instance (VMI) configuration.
+
+
+.Procedure
+
+. Include details of the TCP readiness probe in the VMI configuration file.
++
+
+.Sample readiness probe with a TCP socket test
+[source,yaml]
+----
```

Comment:
(applies to all snippets) according to the OCP docs contribution guide, include ellipses to indicate that the file continues: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Optional, but encouraged: include the beginning of the YAML so that the `kind:` field is visible.

Reply:
Added ellipses at the beginning and end of the code block for all snippets